### DISCUSSION TITLE
Switching to a different version

### DISCUSSION THREAD

**Discussion by msh855** [quarto-dev/quarto-cli#5327] [Q&A] [answered]
Hi, 

I have installed the 1.3.340 of Quarto. Is possible to switch to an earlier version? I cannot find the information for this.

**Comment by mcanouil** [comment, marked as answer]
What does "buggy" mean?
Also what do you mean by "latest"? Stable or pre-release?

All releases can be found on GitHub: <https://github.com/quarto-dev/quarto-cli/releases>.

**Reply by msh855** [reply]
I updated my question. Hope more clear. 

I went through the link. But how do I install an earlier version?

**Reply by mcanouil** [reply]
Choose the release, open it, expand "assets", download the one you need for your operating system and use the standard install process of your operating system.

**Reply by dragonstyle** [reply]
You can also access older releases here:

https://quarto.org/docs/download/

**Reply by mcanouil** [reply]
Indeed easier, if one wants to use the latest patch version for each minor version of Quarto (it still points to the GitHub release page 😏)

**Comment by mcanouil** [comment]
I am still wondering what's the issue you were having that force you to downgrade ... 🤔
